### PR TITLE
improve text for searching prior knowledge

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR changes the text when searching for prior knowledge so that it is more clear this is not yet the screening phase.

### CODE REVIEW

File: asreview/webapp/src/PreReviewComponents/ListItemPaper.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ const ListItemPaper = (props) => {
         fullWidth={true}
       >
         <DialogTitleWithClose
-          title={"Is this article relevant?"}
+          title={"If this document can be used as prior knowledge, is it relevant or irrelevant?"}
```
Comment (Rensvandeschoot) [inline asreview/webapp/src/PreReviewComponents/ListItemPaper.js:88]: @J535D165 What is the maximum of characters I can use?

Comment (terrymyc) [reply asreview/webapp/src/PreReviewComponents/ListItemPaper.js:88]: image

It looks too long as a title...

Comment (Rensvandeschoot) [reply asreview/webapp/src/PreReviewComponents/ListItemPaper.js:88]: @MartijnUX Do you have suggestions?

**Comment by J535D165** [comment]
Thanks, hopefully, this resolves the confusion for some users!

**Pull request merged**